tests/all/readme.rs

```diff
@@ -47,7 +47,7 @@ fn it_creates_a_package_json_provided_path() {
     assert!(fs::metadata(&crate_readme_path).is_ok());
     assert!(fs::metadata(&pkg_readme_path).is_ok());
 
-    let crate_readme = utils::readme::read_file(&crate_readme_path).unwrap();
-    let pkg_readme = utils::readme::read_file(&pkg_readme_path).unwrap();
+    let crate_readme = utils::file::read_file(&crate_readme_path).unwrap();
```

Comment:
It's a bit late now, but `std` has a convenience function that does this now called:

`std::fs::read_to_string` and for bytes `std::fs::read`!

One of my favorite new additions to the standard library